recipes-support/provisioning/esec-provisioning.bb

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2021 Maik Otto <m.otto@phytec.de>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+SUMMARY = "Provisioning Tools for ESEC IOTDM Plattform"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+SRC_URI = " \
+    file://test-provision-tpm.sh \
+"
+
+S = "${WORKDIR}"
+
+do_install() {
+    install -m 0500 test-provision-tpm.sh ${D}/home
+}
+
+FILES_${PN} += "/home"
```

Comment:
It is advisable to explicitly install each file that is needed. In this case only install `/home/test-provision-tpm.sh`.

Reply:
Also there is no need to append using `+=`. Simply setting `FILES` with `=` should do the trick.